src/agent/sqlite_manager/include/sqlite_manager.hpp

```diff
@@ -97,5 +102,6 @@ namespace sqlite_manager
 
         ColumnType ColumnTypeFromSQLiteType(const int type) const;
         void Execute(const std::string& query);
+        std::string m_lastQuery;
```

Comment:
We don't need this. As part of https://github.com/wazuh/wazuh-agent/issues/92 we will improve the logging of the different components.

Reply:
Done.